infrastructure/hosts/catcolab-next/backend.nix

```diff
@@ -183,17 +210,28 @@ in {
     };
 
     security.sudo.extraRules = [{
-        users = [ "catcolab" ]; 
+        users = [ "catcolab" "deployuser" ];
         commands = [
             { command = "/run/current-system/sw/bin/systemctl start automerge"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl stop automerge"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl restart automerge"; options = [ "NOPASSWD" ]; }
             { command = "/run/current-system/sw/bin/systemctl start backend"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl stop backend"; options = [ "NOPASSWD" ]; } 
             { command = "/run/current-system/sw/bin/systemctl restart backend"; options = [ "NOPASSWD" ]; }
+            { command = "${updateScript}/bin/catcolab-update";  options = [ "NOPASSWD" ]; }
         ]; 
     }];
 
+
+    users.users.deployuser = {
```

Comment:
I was unable to put the new deployuser user with the other users in the hosts default.nix without adding a layer of abstraction to the nix configs (there is no easy/clean way to access updateScript from default.nix)